osu.Game.Rulesets.Taiko/Difficulty/TaikoDifficultyAttributes.cs

```diff
@@ -13,21 +13,25 @@ public class TaikoDifficultyAttributes : DifficultyAttributes
         /// <summary>
         /// The difficulty corresponding to the rhythm skill.
         /// </summary>
+        [JsonProperty("rhythm_difficulty")]
         public double RhythmDifficulty { get; set; }
 
         /// <summary>
         /// The difficulty corresponding to the reading skill.
         /// </summary>
+        [JsonProperty("reading_difficulty")]
```

Comment:
Adding these `JsonProperty` attributes back requires them to be databased attributes in order to not cause confusion (see https://github.com/ppy/osu/pull/32354 for details)

Reply:
Didn't catch that specific pr, its intention was yes, to display them back in perfcalc after reports they weren't showing. I'll remove this for now, as we plan to yes, database more attributes, but they will be split into 

`mechanical_difficulty` (colour + stamina)
`reading_difficulty`
`rhythm_difficulty`